src/textual/_border.py

```diff
@@ -5,20 +5,23 @@
 from rich.console import Console, ConsoleOptions, RenderResult, RenderableType
 import rich.repr
 from rich.segment import Segment, SegmentLines
-from rich.style import Style, StyleType
+from rich.style import Style
 
-from .color import Color
 from .css.types import EdgeStyle, EdgeType
 
 
 INNER = 1
 OUTER = 2
 
 BORDER_CHARS: dict[EdgeType, tuple[str, str, str]] = {
-    # TODO: in "browsers' CSS" `none` and `hidden` both set the border width to zero. Should we do the same?
+    # Each string of the tuple represents a sub-tuple itself:
```

Comment:
As I was removing this "TODO" (which is now covered by this PR) I also added a bit of internal documentation in order to understand this data structure :-)